osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -116,7 +118,7 @@ public void ShowUser(User user, bool fetchOnline = true)
 
             if (fetchOnline)
             {
-                userReq = new GetUserRequest(user.Id);
+                userReq = user.Username != null ? new GetUserRequest(user.Username) : new GetUserRequest(user.Id);
```

Comment:
Should probably be the other way around - only use username if `Id <= 1` because `Id` should always be more accurate.

Reply:
Sure, did `Id > 1 ? new GetUserRequest(user.Id)` instead because I think it's slightly more readable